File: CS/SilverlightApplication1/BusyIndicator.cs
namespace DevExpress.Xpf.Toolkit {
    using System;
    using System.Windows.Controls;
    using System.Windows.Media.Animation;
    public class BusyIndicator : Control {
        public BusyIndicator() {
            DefaultStyleKey = typeof(BusyIndicator);
        }
        public override void OnApplyTemplate() {
            base.OnApplyTemplate();
            Storyboard story = (Storyboard)GetTemplateChild("Storyboard");
            if (story != null) {
                story.Begin();
            }
        }
   }
}

File: VB/BusyIndicator.vb
Imports Microsoft.VisualBasic
	Imports System
	Imports System.Windows.Controls
	Imports System.Windows.Media.Animation
Namespace DevExpress.Xpf.Toolkit
	Public Class BusyIndicator
		Inherits Control
		Public Sub New()
			DefaultStyleKey = GetType(BusyIndicator)
		End Sub
		Public Overrides Sub OnApplyTemplate()
			MyBase.OnApplyTemplate()
			Dim story As Storyboard = CType(GetTemplateChild("Storyboard"), Storyboard)
			If story IsNot Nothing Then
				story.Begin()
			End If
		End Sub
	End Class
End Namespace